### PR TITLE
[Features] translation, donors, donations, subscriptions queries, inivitation and welcome feature

### PR DESCRIPTION
**Pull request by jstaricco10**

### CODE REVIEW

File: src/features/donations/single-donation-table/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  const onChangePage = (_event: any, newPage: number) => {
+    setPage(newPage);
+  };
+
+  return (
+    <Grid item xs={12}>
+      <CustomTable
+        data={mappedDonations}
+        columnLabels={['Donador', 'Proyecto', 'Monto', 'Fecha', 'ID de pago']}
```
Comment (jopealgorta) [inline src/features/donations/single-donation-table/index.tsx:53]: DONANTEEEEEEEEEE

File: src/features/team-members/add-team-member-drawer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <FormDrawer
+      isOpen={true}
+      onCloseDrawer={onClose}
+      canSubmit={!isAddDisabled()}
+      onSubmit={onSubmitTeamMember}
+      title='Agregar Miembro'
+      submitButtonText='Agregar'
+      description='Añada un nuevo miembro administrador al equipo de trabajo de su organización. La persona recibira un email con las instrucciones para acceder a la plataforma y realizar su cambio de contraseña.'
+    >
+      <FormInput
```
Comment (jopealgorta) [inline src/features/team-members/add-team-member-drawer/index.tsx]: TextField pleaaaase

Comment (jstaricco10) [reply src/features/team-members/add-team-member-drawer/index.tsx]: Eating ear

Comment (jopealgorta) [reply src/features/team-members/add-team-member-drawer/index.tsx]: jajajajaja

**Review by jopealgorta — APPROVED** [review]
Thank you men! Great work

**Pull request merged**